arangod/RocksDBEngine/RocksDBDumpContext.cpp

```diff
@@ -452,28 +452,34 @@ void RocksDBDumpContext::handleWorkItem(WorkItem item) {
       continue;
     }
 
-    auto storedSlice = [&]() -> VPackSlice {
+    // Sanitize document
+    VPackValueLength const inputLength = documentSlice.byteSize();
+    VPackBuffer<uint8_t> tmpBuffer;
+    bool resolveExt =
+        basics::VelocyPackHelper::hasNonClientTypes(documentSlice);
+    if (resolveExt) {                  // resolve
```

Comment:
You could put the call to `hasNonClientTypes` directly into the `if`, since it does not seem to be used later, so it does not need a name. However, this is a matter of taste and I leave the decision if you do this change entirely to you.